docker-compose.ci.yml

```diff
@@ -18,27 +18,10 @@ x-api:
   expose:
     - 8000
 
-# Define common behavior of worker
-x-worker:
-  &default-worker
-  image: erica_worker:latest
-  build:
-    context: .
-    target: worker
-    args:
-      bucket_name: ${ERICA_BUCKET_NAME}
-      access_key_id: ${ACCESS_KEY_ID}
-      access_key: ${ACCESS_KEY}
-      endpoint_url: ${ENDPOINT_URL}
-      elster_datenlieferant: ${ELSTER_DATENLIEFERANT}
-      elster_hersteller_id: ${ELSTER_HERSTELLER_ID}
-  expose:
-    - 8000
```

Comment:
Why can we get rid of that?

Reply:
No separate worker image. Worker uses the main image.

Reply:
Remember this docker-compose file is only used to build images, never to actually run services.

Reply:
👍